### ISSUE TITLE
strongtyping reimplementation

### ISSUE THREAD

**Issue by mistoo** [djberg96/html-table#4]
Overriding StrongTyping is troublesome for users who actually using it cause a) Ruby implementation is slower than original, b) it's not compatible with original (compare expect signatures) and c) it simply does not work properly with Sequel models:

```
[1] pry(main)> expect(User.new, User)
TypeError: class or module required
from /usr/local/share/gems/gems/html-table-1.6.0/lib/strongtyping.rb:10:in `kind_of?'
[2] pry(main)> 

```
In general, changing other gems it's not good practice in my opinion. Would you consider removing this override? If so, I could prepare PR for this.

**Comment by djberg96** [comment]
Hm, I should probably both fix it to be compatible and namespace it properly to avoid future conflicts. If you have a PR, I'll certain take a look.

**Comment by mistoo** [comment]
No, I haven't PR (I just downgraded to prev version), so feel free to do this ;)

**Comment by djberg96** [comment]
@mistoo How's it look? https://github.com/djberg96/html-table/pull/5

**Comment by djberg96** [comment]
Merged and released in 1.6.1. Thanks for the report!

**Comment by mistoo** [comment]
Great, thank you!
